ballerina-tests/tests/36_subscriptions.bal

```diff
@@ -355,8 +355,19 @@ function testAlreadyExistingSubscriber() returns error? {
     check sendSubscriptionMessage(wsClient, document, clientId);
     check sendSubscriptionMessage(wsClient, document, clientId);
 
+    json|error response;
+    while true {
+        response = readMessageExcludingPingMessages(wsClient);
+        if response is error {
+            break;
+        }
+        json|error id = response.id;
+        if id is error {
+            test:assertFail(string `Expected json with id found: ${response.toString()}`);
+        }
```

Comment:
Can this lead into an infinite loop? What if we don't get an error?

Reply:
Since server side uses multiplexing it will return an error for the second subscription operation

Reply:
Updated fail after 4 iteration